docs/alert-spec.md

```diff
@@ -16,19 +16,23 @@ packs/<pack_name>/alerts/static-alert.yml
 
 ```yaml
 ####################################################################################
-# Metadata - these fields will be removed from the AlertsNrqlConditionInput 
-# definition
+# Metadata - these fields will be removed from the AlertsNrqlConditionInput
 ####################################################################################
 
-# A short form description for this Alert condition
+# the name of the Alert
+name: string, required
+
+# A short description for this Alert condition
 details: string, optional
 
-# Type of alert condition
+# Type of Alert condition
 type: enum, required # One of [ BASELINE, OUTLIER, STATIC ]
 
+# Url of the Alert in this pack
+url: string, required
```

Comment:
What would be the value of this url? I'm not sure what it would point to. Also we may want to avoid arbitrary URLs in these configurations because they can leave you open to XSS attacks.

Reply:
Not sure, it's in Graphql though the comment says URL of Alert JSON definition

<img width="373" alt="Screen Shot 2021-08-19 at 9 00 23 AM" src="https://user-images.githubusercontent.com/1395158/130092223-f6236d7f-2190-4778-805a-422f12151a09.png">

Reply:
`url` doesn't belong in the source alert spec - it's something added by the service and will be ignored if the user supplies it.

Reply:
I'll remove it.